sdcm/tester.py

```diff
@@ -3339,7 +3339,14 @@ def check_latency_during_ops(self, hdr_tags: list[str]):
                                     events=get_events_grouped_by_category(
                                         _registry=self.events_processes_registry))
         with open(self.latency_results_file, encoding="utf-8") as file:
-            latency_results = json.load(file)
+            try:
```

Comment:
Why? This is a pretty huge change

Reply:
this is a small change - just validate a file is opened and json successfully loaded before continuing.
i know it fails some times and it doesn't make sense to fail the whole test because of that.
it might be a good idea to further investigate and fix it in a followup PR.